src/Feature/Navigation/client/GlobalHeader/components/NavUser/index.tsx

```diff
@@ -189,8 +187,7 @@ export default class NavUser extends Jss.SafePureComponent<NavUserProps, NavUser
         formValid: true,
       });
     }
-    const { email, password } = this.state;
 
-    StartAuth(email, password);
+    StartAuth(this.emailRef.current.value, this.passwordRef.current.value);
```

Comment:
Why you use `emailRef` if you have `email` in state?

Reply:
for similar behaviour as password

Reply:
Replace `this.emailRef.current.value` with `email` or use `emailRef` everywhere